java/src/main/java/com/google/oak/crypto/BUILD

```diff
@@ -28,6 +29,13 @@ java_library(
     ],
 )
 
+android_library(
```

Comment:
Does every single dependency need to be an Android library?

Reply:
AFAIK, unfortunately yes.

Our goal is for an `android_library` to depend on oak/client:client. From the [bazel documentation](https://bazel.build/reference/be/android#android_library_args), there are restrictions on what an `android_library` can depend on. In theory (and in those docs) we could have a shared `java_library` with the `android` constraint, but `constraint` isn't available in the `java_library` rule ([context](https://github.com/bazelbuild/bazel/issues/3555)).

Reply:
mitigated by adding android constraint during piper import